index.html

```diff
@@ -0,0 +1,858 @@
+          </p>
+          <p>
+            Confirming inflation would mean that the universe we see, extending
+            14 billion light-years in space with its hundreds of billions of
+            galaxies, is only an infinitesimal patch in a larger cosmos whose
+            extent, architecture and fate are unknowable. Moreover, beyond our
+            own universe there might be an endless number of other universes
+            bubbling into frothy eternity, like a pot of pasta water boiling
+            over.
+          </p>
+
+          <p><strong>‘As Big as It Gets’</strong></p>
+
+          <p>
+            In our own universe, it would serve as a window into the forces
+            operating at energies forever beyond the reach of particle
+            accelerators on Earth and yield new insights into gravity itself.
+            Dr. Kovac’s ripples would be the first direct observation of
+            gravitational waves, which, according to Einstein’s theory of
+            general relativity, should ruffle space-time.
+          </p>
+          <p>
+            Marc Kamionkowski of Johns Hopkins University, an early-universe
+            expert who was not part of the team, said, “This is huge, as big as
+            it gets.”
+          </p>
+          <p>
+            He continued, “This is a signal from the very earliest universe,
+            sending a telegram encoded in gravitational waves.”
+          </p>
+          <p>
+            The ripples manifested themselves as faint spiral patterns in a bath
+            of microwave radiation that permeates space and preserves a picture
+            of the universe when it was 380,000 years old and as hot as the
+            surface of the sun.
+          </p>
+          <p>
+            Dr. Kovac and his collaborators, working in an experiment known as
+            Bicep, for Background Imaging of Cosmic Extragalactic Polarization,
+            reported their results in a scientific briefing at the Center for
+            Astrophysics here on Monday and in a set of papers submitted to The
+            Astrophysical Journal.
+          </p>
+        </div>
+      </div>
+
+      <div class="figcaption">
+        <h2>The Theory of Inflation</h2>
+        <p>
+          Astronomers have found evidence to support the theory of inflation,
+          which explains how the universe expanded so uniformly and so quickly
+          in<br />
+          the instant after the Big Bang 13.8 billion years ago.
+        </p>
+      </div>
+
+      <div class="theoryInflation">
+        <div class="row1">
+          <div class="column">
+            <p>
+              THE UNIVERSE is just under 14 billion years old. From our position
+              in the Milky Way galaxy, we can observe a sphere that is now about
+              92 billion light-years across. But there's a mystery. Wherever we
+              look, the universe has an even temperature.
+            </p>
```

Comment:
This 'Theory of Inflation' section should not occupy the full width as mentioned by @christianotieno

Reply:
I Have improved this and it is no longer occuply full section in new current version

![image](https://user-images.githubusercontent.com/70282171/108470666-fb30c080-7292-11eb-9e87-8b0681aa3079.png)